src/Level.h

```diff
@@ -0,0 +1,27 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2021 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <string>
+#include <filesystem>
+
+namespace openblack
+{
+class Level
+{
+public:
+	Level(std::filesystem::path path) : _name(path.filename().string()), _scriptPath(path) {}
```

Comment:
You could move path instead of taking a copy

Reply:
So path is an r-value reference?

Reply:
Using std::move from the caller